lib/widgets/shimmers/search_initial_page_shimmer_widget.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:Medito/widgets/shimmers/widgets/box_shimmer_widget.dart';
+import 'package:Medito/constants/constants.dart';
+import 'package:flutter/material.dart';
+
+class SearchInitialPageShimmerWidget extends StatelessWidget {
+  const SearchInitialPageShimmerWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      padding: EdgeInsets.only(top: 20, left: 15, right: 15),
```

Comment:
15 should probably be 16 here. Can you check please? They will 99% of the time be multiples of 4

Reply:
Yeah its 16px. If you use the inspect mode on figma you can see the padding is 16px :) thanks both